client/state/data-layer/wpcom-http/actions.js

```diff
@@ -35,7 +37,7 @@ export const http = ( {
 	body,
 	method,
 	path,
-	query: { ...query, apiVersion },
+	query: { ...query, apiVersion, apiNamespace },
```

Comment:
Yes, there is some documentation in wpcom request handlers but to be honest it's a little poor. I'll try to find something about this.